porting/common/src/sysinit/sysinit.c

```diff
@@ -34,7 +36,8 @@ sysinit_dflt_panic_cb(const char *file, int line, const char *func,
     }
 #endif
 
-    __assert_func(file, line, func, expr);
+    __assert_fail(msg, file, line, func);
+    //__assert_func(file, line, func, expr);
```

Comment:
This is a known conflict between FreeRTOS and Linux that needs to be resolved before merging.